js/zb.js

```diff
@@ -716,12 +716,102 @@ module.exports = class zb extends Exchange {
         return this.safeBalance (result);
     }
 
+    parseSwapBalance (response) {
+        const timestamp = this.milliseconds ();
```

Comment:
let's not assign the timestamp from `this.milliseconds` also timestamp isn't a field in `fetchBalance`